### PR TITLE
Updated some text

### PR DESCRIPTION
**Pull request by murshed**
Updates some files

**Description**
I have updated the theme description.

### CODE REVIEW

File: assets/css/button-outline.min.css [hunk 1]
```diff
@@ -0,0 +1 @@
+.wp-block-button.is-style-outline>.wp-block-button__link:not(.has-text-color,.has-background):hover{background-color:var(--wp--preset--color--contrast-2,var(--wp--preset--color--contrast,transparent));color:var(--wp--preset--color--base);border-color:var(--wp--preset--color--contrast-2,var(--wp--preset--color--contrast,currentColor))}
```
Comment (MaggieCabrera) [inline assets/css/button-outline.min.css:1]: Is this change intended?

Comment (murshed) [reply assets/css/button-outline.min.css:1]: If not required, you can disregard.

**Comment by MaggieCabrera** [comment]
Closed in favor of https://github.com/WordPress/twentytwentyfour/pull/671
